ortho_seq_code/orthogonal_polynomial.py

```diff
@@ -92,7 +92,7 @@ def orthogonal_polynomial(
                 phi[j][i][2] = 1.0
 
     naming = os.path.basename(f.name)
-    if precomputed:
+    if precomputed == 'True':
```

Comment:
this is not how you compare a boolean if precomputed already does that you can't compare a boolean to a string called True!

Reply:
I am changing it back, please pull @snafees